brainunit/__init__.py

```diff
@@ -31,6 +32,8 @@
 from ._unit_constants import __all__ as _constants_all
 from ._unit_shortcuts import *
 from ._unit_shortcuts import __all__ as _std_units_all
+from .lax import *
```

Comment:
we should not import functions in `brainunit.lax` into ``brainuni``'s namespace.